fontstash/fontstash.h

```diff
@@ -479,10 +490,10 @@ struct FONSshapingRes
 typedef struct FONSshapingRes FONSshapingRes;
 
 struct FONSshaping {
-    FONSshapingRes* shapingRes;
-    const char* script;
-    const char* direction;
-    const char* language;
+    FONSshapingRes* result;
+    void* script;
+    void* direction;
+    void* language;
     int it;
 };
```

Comment:
I didn't want to restrict the possibility, but since the shaping engine choice is pretty restricted.. :) Anyway you're right; considering the size of a shaping implementation we could easily use the Harfbuzz types here.
What do you think of typedefing those from one implementation to keep things this way?